schemas/v1.0/azure.yaml.json

```diff
@@ -27,6 +27,31 @@
                 }
             }
         },
+        "infra": {
+            "type": "object",
+            "additionalProperties": true,
+            "properties": {
+                "provider":{
+                    "type": "string",
+                    "title": "Type of infrastructure provisioning provider",
+                    "description": "Optional. When specified will set the infrastructure provisioning provider used to provision the Azure resources for the application. (Default: bicep)",
+                    "enum": [
+                        "bicep",
+                        "terraform"
+                    ]
+                },
+                "path": {
+                    "type": "string",
+                    "title": "Path to the location that contains Azure provisioning templates",
```

Comment:
Does this support both relative and full path?

Reply:
Maybe the same question can be imposed: do we think this is a useful enough feature to break convention? Being opinionated about repo structure might not be a bad thing.

Reply:
It is relative paths.  Will update the wording to call this out.